package.json

```diff
@@ -24,9 +24,21 @@
     },
     "dependencies": {
         "@nestjs/common": "^8.0.0",
+        "@nestjs/config": "^1.0.1",
         "@nestjs/core": "^8.0.0",
+        "@nestjs/jwt": "^8.0.0",
+        "@nestjs/passport": "^8.0.0",
         "@nestjs/platform-express": "^8.0.0",
         "@nestjs/typeorm": "^8.0.1",
+        "@types/bcrypt": "^5.0.0",
+        "bcrypt": "^5.0.1",
+        "class-transformer": "^0.4.0",
+        "class-validator": "^0.13.1",
+        "cookie-parser": "^1.4.5",
```

Comment:
remove additional packages.

Reply:
We need @nestjs/passport and passport both and apart from that based on the strategy that we need to implement
<img width="797" alt="passportjs" src="https://user-images.githubusercontent.com/87020866/127139851-1f9c94e8-a8e2-47cf-8b2b-b490c0a97111.png">